flash_examples/finetuning/image_classification.py

```diff
@@ -27,16 +30,30 @@
     test_folder="data/hymenoptera_data/test/",
 )
 
-# 3. Build the model
-model = ImageClassifier(num_classes=datamodule.num_classes)
 
-# 4. Create the trainer.
+# 3. a Register a custom backbone
```

Comment:
```suggestion
# 3.a Register a custom backbone
```

Also, should mention that this is optional

Reply:
Done !